sqlglot/optimizer/qualify_columns.py

```diff
@@ -277,6 +283,27 @@ def _get_unambiguous_columns(source_columns):
     return unambiguous_columns
 
 
+def _get_scope_schema_columns(source_columns):
+    """
+    Get all the schema columns from sources.
+
+    Args:
+        source_columns (dict): Mapping of names to source columns
+    Returns:
+        list: all columns from schema
+    """
+    if not source_columns:
+        return []
+
+    columns = [
+        value
+        for nested_list_value in list(source_columns.values())
```

Comment:
the list is unnecessary

Reply:
can just do

return [
   column
   for columns in source_columns.values()
   for column in columns
]

Reply:
Ow yes. Indeed.

Reply:
i'll fix it